variables.tf

```diff
@@ -104,8 +104,8 @@ variable "nats" {
 
     subnetworks = optional(list(object({
       name                     = string
-      source_ip_ranges_to_nat  = string
-      secondary_ip_range_names = optional(string)
+      source_ip_ranges_to_nat  = set(string)
```

Comment:
can you plz change it to list instead of set?

Reply:
Changed it to a list. However the provider does expect a set:

```
│ Inappropriate value for attribute "source_ip_ranges_to_nat": set of string
```